### PR TITLE
Ensure test resets logging after changing it.

### PR DESCRIPTION
**Pull request by rnc**

### CODE REVIEW

**Comment by rnc** [comment]
retest this please

**Comment by rnc** [comment]
retest this please

File: src/test/java/com/redhat/red/build/finder/BuildFinderTest.java [hunk 1]
```diff
@@ -36,13 +38,19 @@
...
-        BuildFinder.main(new String[] {"-d", "-k", "-o", folder.getAbsolutePath(), target.getAbsolutePath()});
+            File target = new File(TestUtils.resolveFileResource("./", "").getParentFile().getParentFile(), "pom.xml");
+            BuildFinder.main(new String[] {"-d", "-k", "-o", folder.getAbsolutePath(), target.getAbsolutePath()});
 
-        assertTrue(systemOutRule.getLog().contains("DEBUG"));
+            assertTrue(systemOutRule.getLog().contains("DEBUG"));
+        } finally {
+            final ch.qos.logback.classic.Logger root =
+                (ch.qos.logback.classic.Logger) LoggerFactory.getLogger(org.slf4j.Logger.ROOT_LOGGER_NAME);
+            root.setLevel(Level.INFO);
```
Comment (dwalluck) [inline src/test/java/com/redhat/red/build/finder/BuildFinderTest.java]: Doesn't this line assume that the previous level was set to `INFO`? Shouldn't it get the previous level programmatically?

Also, isn't this a problem in the `BuildFinder` code itself not resetting the log level as opposed to the `BuildFinderTest` code here?

Comment (rnc) [reply src/test/java/com/redhat/red/build/finder/BuildFinderTest.java]: As its running in the same JVM the logging level is effectively global. I concur we could have obtained it programmatically .... currently the surefire tests run at info level logging. If that is changed we could improve this.

**Pull request merged**